Justfile

```
@@ -132,7 +135,7 @@ dev-host-host-agent-remote-hc desired-status +args="":
     set -xeE
     export NATS_URL="nats://dev-host"
     just host-agent-remote-hc {{desired-status}} \
-        --host-id "{{HOST_ID_DEV_HOST}}" \
+        --host-id "{{HOST_ID_DEV_HOST0}}" \
```

```suggestion
        --host-id "{{HOST_ID_DEV_HOST0}}" \
```

either need to look up the object id in the orchestrator (or hardcode it for the test hosts?) or make the command do this internally